src/components/Editor/GeneralEditor.test.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-this-alias */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GeneralEditor } from './GeneralEditor';
+import { VideoEditorToolbar } from './Editors/VideoEditorToolbar';
+
+global.document.execCommand = jest.fn((value) => value);
```

Comment:
what is this for?

Reply:
> what is this for?

the test can't run because execCommand (it's a part of library) is not a function. So I mocked it.